prow/jira/fakejira/fake.go

```diff
@@ -57,6 +58,24 @@ func (f *FakeClient) GetIssue(id string) (*jira.Issue, error) {
 	return nil, jiraclient.NewNotFoundError(fmt.Errorf("No issue %s found", id))
 }
 
+type GetIssueRequest struct {
+	issueID string
+	options *jira.GetQueryOptions
```

Comment:
We should not use pointers as part of the request struct. We should store `options` as a non-pointer variable and then dereference the pointers given to use in the `GetIssueWithOptions` function. If we use pointers as part of this struct (which we then use as a key in `GetIssueResponses`), this won't work properly because it will compare the pointer value, not the data in the struct.

Reply:
Fixed